cnv/cnv-release-notes/cnv-release-notes.adoc

```diff
@@ -196,4 +160,32 @@ running either `oc adm drain` or `kubectl drain`. Do not run these commands on
 the nodes of any clusters where {CNVProductName} is deployed. The nodes might not
 drain if there are virtual machines running on top of them.
 The current solution is to put nodes into maintenance.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1707427[*BZ#1707427*])
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1795182[*BZ#1795182*])
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1795186[*BZ#1795186*])
+
+* When running `virtctl image-upload` to upload large VM disk images in `qcow2`
```

Comment:
OK, since we may need to get this merged sooner than later, we'll move ahead but we can always update later if we hear otherwise.... Assuming you'll follow up with Natalie, but if you want me to, let me know.

Reply:
i believe Natalie marked it as verified

Reply:
I saw that. Thanks.